tests/group_functions/group_joint_reduce.cpp.in

```diff
@@ -20,32 +20,31 @@
 
 #include "group_reduce.h"
 
+// clang-format off
+#cmakedefine CTS_TYPE @CTS_TYPE@
+#cmakedefine CTS_TYPE_NAME std::string("@CTS_TYPE_NAME@")
+// clang-format on
 using ReduceTypes = Types;
 
-// 2-dim Cartesian product of type lists
-using prod2 = product<std::tuple, ReduceTypes, ReduceTypes>::type;
-
 // hipSYCL has no implementation over sub-groups
```

Comment:
I suggest we remove this and similar comments, where tests are **not** disabled for `AddaptiveCPP` implementation.

Reply:
Can we move the comment to the place in sources where it's disabled?

Reply:
Sure, I have fixed in [8e91479](https://github.com/KhronosGroup/SYCL-CTS/pull/815/commits/8e91479f35d8ee1043f33d4692a43d3f4c385b66)
Comment 1 has been added [here](https://github.com/KhronosGroup/SYCL-CTS/blob/57cc89debffda5c81b0e3f95f2ee9b5d63c7dade/tests/group_functions/group_reduce.h#L154). Comment 2 looks invalid now.